src/providers/UserProvider.tsx

```diff
@@ -11,37 +12,34 @@ export type UserProviderProps = {
 
 export default function UserProvider({ children }: UserProviderProps): JSX.Element {
   const [userState, setUserState] = useRecoilState(userAtom);
-  const [userPreferences, setUserPreferences] = useState<PreferencesType>({});
+  const [user, setUser] = useState<User>();
+  const [userPreferences, setUserPreferences] = useState<PreferencesType>();
+
+  const updateUserPreferences = useCallback(async (preferences: PreferencesType) => {
+    const response = await PreferencesService.updateUserPreferences(preferences);
+    if (response.requestSucceeded) {
+      setUserPreferences(response.preferences ?? {});
+    }
+
+    return Promise.resolve(response.requestSucceeded);
+  }, []);
 
```

Comment:
exposing this so user preference can be updated.. and patched back into the provided data for downstream consumption. currently only used by the organization provider but will expand the usage across code after this fix is deployed